elements/virtual-waggle/post-install.d/03-beehive-server

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+if [ \${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+mkdir -p /opt/beehive-repo
+
+cd /opt/beehive-repo
+
+git clone https://github.com/waggle-sensor/beehive-server.git .
+
+cd /opt/beehive-repo
+
+echo "BEEHIVE_REPO=/opt/beehive-repo" >> /etc/environment
+echo "BEEHIVE_ROOT=/opt/beehive-server" >> /etc/environment
+
+cat > /usr/local/sbin/beehive-server  <<EOF
```

Comment:
I'd suggest a similar thing here as to virtual-waggle. Instead of duplicating, just do an alias.